pallets/ds-maps/src/lib.rs

```diff
@@ -0,0 +1,392 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+use frame_support::{
+    codec::{Decode, Encode},
+    storage::StorageDoubleMap,
+    sp_runtime::sp_std::ops::{Add, Sub, Div, Mul, Shl, BitOr},
+    decl_error, decl_event, decl_module, decl_storage, dispatch, ensure,    
+    weights::Weight,
+    Parameter,
+};
+use substrate_fixed::types::U9F23;
```

Comment:
Since the coordinate type is parameterized, it is highly unlikely that we ever need the concrete type U9F23 anywhere in this crate.
The concrete type must only be used during instantiation of the pallet - in the runtime.